src/landingpage/en-US/uproxy.html

```diff
@@ -0,0 +1 @@
+<h1>WELCOME TO UPROXY!</h1>
```

Comment:
Wait, is this the landing page that goes into C:/Program Files/Mozilla Firefox/uproxy.html?